docs-generator/pkg/crd/crd.go

```diff
@@ -100,11 +101,44 @@ func (c *CRDer) Validate(data []byte) error {
 	return nil
 }
 
+// Remove null values from enums
+//
+// kube-rs 2.0.1 appends a null to all enums which cannot be processed by this module.
```

Comment:
```suggestion
// kube-rs 2.0.1 appends a null to all enums. These null values cannot be processed by this module.
```

Or add a comma after "enums"